### PR TITLE
Feature/register users in cmr2

### PR DESCRIPTION
**Pull request by smartsena**

### CODE REVIEW

File: support/views.py [hunk 1]
```diff
@@ -18,9 +18,13 @@ class AuthModelMixIn:
 class LayersGroupView(generics.ListAPIView, AuthModelMixIn):
     """ Layers Group data view """
 
-    queryset = LayersGroup.objects.all()
+    lookup_field = 'id'
```
Comment (dagnaldo) [inline support/views.py:21]: Please, implement `DjangoFilterBackends`

**Review by dagnaldo — CHANGES_REQUESTED** [review]
Please, implement `DjangoFilterBackends `

File: support/filters.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from django_filters import rest_framework
+
+from support import models
+
+
+class LayersGroupFilter (rest_framework.FilterSet):
+    """LayersGroupFilter data.
+
+    Filters:
+        category (int): catagoreis groups list
```
Comment (dagnaldo) [inline support/filters.py:10]: categoreis?

**Review by dagnaldo — CHANGES_REQUESTED** [review]
Docstrings error.

File: support/views.py (hunk 1 above)
Comment (smartsena) [reply support/views.py]: Implemented Django Filter

File: support/filters.py (hunk 2 above)
Comment (smartsena) [reply support/filters.py:10]: Correction completed and APP-support/views.py fixed imports.

Comment (dagnaldo) [reply support/filters.py:10]: still categoreis...

File: support/views.py [hunk 3]
```diff
@@ -1,29 +1,41 @@
 from rest_framework import generics
+from django_filters.rest_framework import DjangoFilterBackend
+
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.authentication import (
```
Comment (dagnaldo) [inline support/views.py:5]: Same here with imports.

**Review by dagnaldo — CHANGES_REQUESTED** [review]
Look the comments.

File: support/views.py (hunk 3 above)
Comment (smartsena) [reply support/views.py:5]: Correction completed.

File: support/views.py [hunk 4]
```diff
@@ -1,29 +1,41 @@
...
+    serializers,
+    filters as support_filters
+)
 
 
 class AuthModelMixIn:
-    """ AuthModelMixIn default class for `support.views` """
+    """AuthModelMixIn default class for `support.views`."""
 
     authentication_classes = (SessionAuthentication, BasicAuthentication)
```
Comment (dagnaldo) [inline support/views.py:20]: And here, use `authentication_classes = (authentication.SessionAuthentication, authentication.BasicAuthentication)`

File: support/views.py [hunk 5]
```diff
@@ -1,29 +1,41 @@
...
     permission_classes = (IsAuthenticated, )
```
Comment (dagnaldo) [inline support/views.py:21]: `permission_classes = (permission.IsAuthenticated, )`

File: support/views.py [hunk 6]
```diff
@@ -1,29 +1,41 @@
...
 
 
 class LayersGroupView(generics.ListAPIView, AuthModelMixIn):
-    """ Layers Group data view """
+    """ Layers Group data view.
```
Comment (dagnaldo) [inline support/views.py:25]: Remove space between `"""` and `Layers`.

**Review by dagnaldo — CHANGES_REQUESTED** [review]
Check requested changes.

File: support/filters.py (hunk 2 above)
Comment (smartsena) [reply support/filters.py:10]: ok

File: support/views.py (hunk 4 above)
Comment (smartsena) [reply support/views.py:20]: ok

File: support/views.py (hunk 5 above)
Comment (smartsena) [reply support/views.py:21]: ok

File: support/views.py [hunk 7]
```diff
@@ -1,29 +1,41 @@
...
 
 
 class AuthModelMixIn:
-    """ AuthModelMixIn default class for `support.views` """
+    """AuthModelMixIn default class for `support.views`."""
 
-    authentication_classes = (SessionAuthentication, BasicAuthentication)
-    permission_classes = (IsAuthenticated, )
+    authentication_classes = (
+        authentication.SessionAuthentication, authentication.BasicAuthentication
```
Comment (dagnaldo) [inline support/views.py:19]: PEP8 formatting error, line has more than 80 chars

**Review by dagnaldo — CHANGES_REQUESTED** [review]
Check it.

File: support/views.py (hunk 7 above)
Comment (smartsena) [reply support/views.py:19]: ok

**Review by dagnaldo — APPROVED** [review]
Elementary my dear Watson...
At this time, I'll approve it.

**Pull request merged**